src/pages/TabHome.tsx

```diff
@@ -75,6 +75,135 @@ const InfoButton = (props: InfoButtonProps) => {
   );
 };
 
+interface SelectCalendarProps {
+  setIsEditCalendar: (newIsOpen: boolean) => void;
+}
+
+const ViewCalendar = (props: SelectCalendarProps) => {
+  const { t } = useTranslation();
+  const { cycles } = useContext(CyclesContext);
+
+  const lastPeriodDays = getLastPeriodDays(cycles);
+
+  return (
+    <IonDatetime
+      style={{ borderRadius: "20px" }}
+      color="light"
+      presentation="date"
+      locale={getCurrentTranslation()}
+      size="cover"
+      firstDayOfWeek={1}
+      highlightedDates={(isoDateString) => {
+        if (cycles.length === 0) {
+          return undefined;
+        }
+
+        const date = startOfDay(parseISO(isoDateString));
+        if (isSameDay(date, startOfToday()) && isForecastPeriodToday(cycles)) {
+          return {
+            backgroundColor: "rgba(var(--ion-color-light-basic-rgb), 0.4)",
+          };
+        } else if (isForecastPeriodDays(date, cycles)) {
+          return {
+            textColor: "var(--ion-color-dark-basic)",
+            backgroundColor: "rgba(var(--ion-color-light-basic-rgb), 0.4)",
+          };
+        } else if (lastPeriodDays.includes(isoDateString)) {
+          return {
+            textColor: "#43348d",
```

Comment:
Mif of hex color and rgba color

Reply:
Yes, because an rgba color is an already created and often used color, but with a given ambiguity. hex color is a color that is used once only here and is selected specifically for this situation. That's why I didn't add it to the styles